drivers/hmis_external_apis/app/assets/javascripts/external_forms.js

```diff
@@ -118,11 +137,69 @@ $(function () {
   });
 });
 
+window.addHouseholdTypeListener = function (individualOrFamilySelector) {
+  // Generates a "household id" that is unlikely to have collisions among form submissions,
+  // without using external library (uuid) or modern browser features (crypto)
+  var generateHouseholdId = function () {
+    var current = Date.now().toString(); // OK? https://caniuse.com/mdn-javascript_builtins_date_now
```

Comment:
can use Date.getTime()?

Reply:
that does seem to have better coverage, thanks